src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -123,11 +133,18 @@
     acc))
 
 (defn render-parsed-text
-  [{:keys [content] :as message-data}]
-  (reduce (fn [acc e]
-            (render-block message-data acc e))
-          [:<>]
-          (:parsed-text content)))
+  [{:keys [content chat-id]
+    :as   message-data}]
+  (let [{:keys [chat-type community-id]} (rf/sub [:chats/chat chat-id])
```

Comment:
This is subscribing to the whole chat object, which means that a rerender will be triggered if any of the fields in `chat` changes.

Better to subscribe only to the state you need, you can have a subscription:

`(let [use-status-tag? (rf/sub [:chats/is-community-chat? chat-id])]`
which re-uses `chats/chat`.

This should never realistically trigger a re-render since the type of a chat never changes.